ntoskrnl/mm/ARM3/pagfault.c

```diff
@@ -78,16 +78,13 @@ MiCheckForUserStackOverflow(IN PVOID Address,
     /* This is where the stack will start now */
     NextStackAddress = (PVOID)((ULONG_PTR)PAGE_ALIGN(Address) - GuranteedSize);
 
-    /* Do we have at least one page between here and the end of the stack? */
-    if (((ULONG_PTR)NextStackAddress - PAGE_SIZE) <= (ULONG_PTR)DeallocationStack)
+    /* Is not there enough additional memory between here and the end of the stack? */
+    if (((ULONG_PTR)NextStackAddress - GuranteedSize) <= (ULONG_PTR)DeallocationStack)
```

Comment:
this is more than a comment change

Reply:
https://github.com/reactos/reactos/blob/c4e273e359584bcf092f9a07ecd10853926722b9/ntoskrnl/mm/ARM3/pagfault.c#L36
typo GuranteedSize > GuaranteedSize

Reply:
It's variable name, not a typo. See #1070.

Reply:
This is not an improvement for the comment, you change a well formed sentence in something that is not a well formed sentence.

Reply:
> this is more than a comment change.. not saying its correct or incorrect

Not really: it is a trivial copypasta fix, as currently `GuaranteedSize = PAGE_SIZE;`.

> This is not an improvement for the comment

Partly reverted.
NB: Simply suggest a better wording, when my English is not good enough.